gocd/templates/bash/check-sentry-errors.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# TODO: Make this work with s4s
+/devinfra/scripts/checks/sentry/release_error_events.py \
+  --project-id="${SENTRY_PROJECT_ID}" \
+  --project-slug="${SENTRY_PROJECT}" \
+  --release="relay@${GO_REVISION_GETSENTRY_REPO}" \
+  --duration=5 \
+  --error-events-limit="${ERROR_LIMIT}" \
```

Comment:
IMO the best would be to check for a _change_ in the number of error events. Would that be feasible?

Reply:
While this is definitely feasible, it would require some additional work to get implemented. This could be a good candidate for a future improvement we make. How does that sound?

Reply:
Sounds good!